heroku/cli
# migrate(W-14170430): Upgrade pg:backups:restore

This PR moves `pg:backups:restore` to the CLI and upgrades to TypeScript

### Test instructions
1. checkout this branch and run `yarn && yarn build`
2. Using an app with a PG backup, run `./bin/run pg:backups:restore -a APP_WITH_PG_BACKUP`
3. Observe the output is the same as the existing cli
